specification/maps/data-plane/Render/preview/2.1/render.json

```diff
@@ -56,7 +56,7 @@
     "/map/tile": {
       "get": {
         "description": "**Applies to**: S0 and S1 pricing tiers.\n\nThe Get Map Tiles API allows users to request map tiles in vector or raster formats typically to be integrated  into a map control or SDK. Some example tiles that can be requested are Azure Maps road tiles, real-time  Weather Radar tiles or the map tiles created using [Azure Maps Creator](https://aka.ms/amcreator). By default,  Azure Maps uses vector tiles for its web map control (Web SDK) and Android SDK.",
-        "operationId": "RenderV2_GetMapTilePreview",
+        "operationId": "GetMapTilePreview",
```

Comment:
Not a preview anymore

Reply:
Renaming this one now will cause method name conflicts since 1.0 and 2.1 will both have GetMapTile method. Autorest failed for generating SDK.

Reply:
Renamed as GetMapTileV2 for now to prevent conflict